enterprise/server/cmd/executor/BUILD

```diff
@@ -88,18 +88,18 @@ go_binary(
     embed = [":executor_lib"],
 )
 
+alias(
+    name = "tini",
+    actual = select({
+        "@platforms//cpu:arm64": "@com_github_krallin_tini_tini-linux-arm64//file:tini",
+        "//conditions:default": "@com_github_krallin_tini_tini-linux-amd64//file:tini",
```

Comment:
does this break the build on mac? or do we not even build oci runtime there

Reply:
yeah, this target is not in the build graph for macs.